spec/views/admin/appropriate_bodies/show.html.erb_spec.rb

```diff
@@ -5,24 +5,23 @@
   before do
     assign(:appropriate_body, appropriate_body)
     assign(:current_ect_count, current_ect_count)
+    render
   end
 
   it 'sets the main heading and page title to the appropriate body name' do
-    render
-
     expect(view.content_for(:page_title)).to start_with(appropriate_body.name)
     expect(view.content_for(:page_header)).to have_css('h1', text: appropriate_body.name)
   end
 
   it 'displays a count of current ECTs' do
-    render
-
     expect(rendered).to have_css('.govuk-summary-list__value', text: current_ect_count)
   end
 
-  it "displays a link to the appropriate body's ECTs" do
-    render
+  it 'links to appropriate body timeline' do
+    expect(rendered).to have_link('View activity', href: admin_appropriate_body_timeline_path(appropriate_body))
```

Comment:
Instead of view activity, could it be view timeline to be more consistent with the H1 on the actual page + how we refer to this generally?

Reply:
"Timeline of events" maybe.